app/js/lib/foxbox.js

```diff
@@ -327,6 +326,25 @@ export default class Foxbox extends Service {
     });
   }
 
+  performServiceCommand(id, command, method = 'GET', state = undefined) {
```

Comment:
I don't think we should do that.
The goal of this lib is to provide an abstraction layer over the communication protocol. I think we need to implement methods like `getSnapshot` or `getVideoStream` that accept the service ID as a parameter and that return whatever the view needs.